arangod/Aql/AqlFunctionFeature.cpp

```diff
@@ -435,6 +435,10 @@ void AqlFunctionFeature::addMiscFunctions() {
   add({"DECODE_REV", ".", flags, &Functions::DecodeRev});
   add({"V8", ".", Function::makeFlags(FF::Deterministic, FF::Cacheable)});  // only function without a
                                                                             // C++ implementation
+                                                                            //
+  auto validationFlags = Function::makeFlags(FF::None);
+  add({"GET_SCHEMA", ".", validationFlags, &Functions::GetValidation});
```

Comment:
I think we should follow the convention and name the C++ function the same as the AQL function, i.e.
```suggestion
  add({"GET_SCHEMA", ".", validationFlags, &Functions::GetSchema});
```